index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio project</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header>
+        <div class="header-container">
+            <h1>My Logo</h1> 
+            <img src="./images/menu.png" width="40" height="40">
+        </div>
+    </header>
+    <section>
+        <div>
+            <h1 class="header">
+                I’m Htetaung to see you!
+            </h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <a href="">LET’S CONNECT</a>
+            <div class="image">
+                <img src="./images/Vector.png">
+                <img src="./images/Vector4.png">
+                <img src="./images/Vector1.png">
+                <img src="./images/Vector2.png">
+                <img src="./images/Vector3.png">       
```

Comment:
- [ ] Kindly make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) 🙏 . Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons
Example: `<li><a><img></a></li>`